### ISSUE TITLE
GradCam

### ISSUE THREAD

**Issue by chrisdottel** [isarandi/metrabs#43]
I'd love to try and implement GradCam on this project, however the pretrained models seem to be packed with the Tensorflow save instead of Keras.  Do you have the checkpoints available for the evaluation or training code for finetuning/ custom implementations?

**Comment by isarandi** [comment]
Do you necessarily need a Keras-style checkpoint for this? It should be possible to backprop through the TF SavedModel as well and get gradients wrt the input image. You may want to use the model.crop_model for this analysis, which is the single-person pose estimator at the core of the method, which takes a 256x256 cropped image. (See bottom of https://github.com/isarandi/metrabs/blob/master/docs/INFERENCE_GUIDE.md)
